src/main/kotlin/team/sfe/server/domain/user/domain/User.kt

```diff
@@ -22,6 +27,17 @@ class User(
 
     @Enumerated(EnumType.STRING)
     @Column(columnDefinition = "VARCHAR(5)")
-    val authority: Authority
+    val authority: Authority,
 
+    @OneToOne
+    @JoinColumn(name = "id")
+    val disease: Disease?,
+
+    @OneToOne
+    @JoinColumn(name = "id")
+    val gameInfo: GameInfo?,
+
+    @OneToOne
+    @JoinColumn(name = "id")
+    val person: Person?
```

Comment:
```suggestion
    @OneToOne
    @JoinColumn(name = "id")
    val disease: Disease? = null,

    @OneToOne
    @JoinColumn(name = "id")
    val gameInfo: GameInfo? = null,

    @OneToOne
    @JoinColumn(name = "id")
    val person: Person? = null
```
여기서 default 값을 설정하고, service에서 값을 안넘기는게 좋을 것 같습니다!